src/seedu/addressbook/AddressBook.java

```diff
@@ -202,12 +202,7 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
-        }
+        readUserCommandsUntilExit();
```

Comment:
you can think of a name that conveys what the method does. for instance here it is not just "read" but it processes the command, displays the output, etc.,